### PR TITLE
Pull Request JS-WeatherApp

### PR DESCRIPTION
**Pull request by alexisbec**
This is a simple Weather App made with Javascript, HTML & CSS, and the OpenWeather API. In the app, you will be able to search any city and see their temperatures in Celsius centigrades.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+To get a local copy up and running follow these simple example steps:
+
+- Clone this repository with:
+```
+  https://github.com/alexisbec/JS-WeatherApp.git
+```
+- Open the project folder where the repository was downloaded.
+- Open the *index.html* file with your live server.
+- Start the navigation throw the site using the tabs on the navbar.
```
Comment (SajjadAhmad14) [inline README.md]: The `npm install` command should be run after cloning the project.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [x] PR title and summary are descriptive.
- [x] Linter checks run perfectly.
## To Improve 🙏
- [x] Please add `npm install` in the `Getting Started` section of README. It should be the first and must to do step in order to run the project locally.
- [x] App should have a toggle button that allows a user to convert Celsius to Fahrenheit and vice versa. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png).
- [x] Please use different modules and do not put all of the code in one file. For example, there can be one module for data fetch from API and one for showing it on screen. Moreover, try to rename bundle file in dist directory and it should not be index.js for the sake of readability. 
- [x] App should change its appearance based on the received data. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png).
- [x] Please see inline comments for reference.
# Optional
- [ ] Deploy the app to Heroku or Netlify and add a live link in README.
### Happy Coding ☺️
## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

@alexisbec Weldone for working on this project,kindly implement all what the previous Code Reviewer suggested,however changes are required
### highlight
Please add npm install in the Getting Started section of README. 👍🏻 

### Improve
- [x] It has a toggle control to determine if the result will be shown in Celsius or Fahrenheit _(There are two valid solutions: students can make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit)_ ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x] It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [ ]  Please use different modules and do not put all of the code in one file. 

##  once specific changes made kindly request for another review

**Comment by alexisbec** [comment]
I compare other approved projects and they had just on js file, also it's not a requirement to have different modules in different files, as the app is working correctly.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, Thank you for working on changes suggested previously :ok_hand:
## To Improve 🙏
- [ ] App should change weather data between Farenheight and Celsius without any hurdle. User when first selects Celsius, can not toggle to Farenheight or vice versa. Please see this screenshot:
screenshot

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Comment by alexisbec** [comment]
Every time that you look for a city the form will be reset. That's why the radios are inside the form, and before the search button.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change Required❌

# 🙌 @alexisbec 

#### You have done a good job, so far, the App is almost perfect, however the last recommendation made in the previous review have not be addressed.

* [ ] What the reviewer sought to say is that you are required to implement the application such that user can view the temperature in either `Degree Celsius` or `Fahrenheit`.
Make sure the unit(symbol) displayed correspond to the unit selected.

A common scenario is that which i think would be more user friendly:

> user enter city name, 
> user click enter then user see temperature in Degree Celsius `temp
> user then click a toggle button that convert the temperature from one unit to the other.
> User see temperature in other temperature either than the previous.
> Here you get the temperature in one unit and use the appropriate conversion formula  to toggle unit back and forth

#### That is kindly make the change and submit a re-review🙋‍♂️

Happy coding

Addo Forison

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The app runs just fine.
    2. The toggle functionality is now set, good job. 👍🏽 

### **_Room for Improvement_** 🤪 

    1. Specify where exactly to find the `index` file to run to get to the app.

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**